pkg/tokens/plugin.go

```diff
@@ -101,6 +101,9 @@ type TokenPool struct {
 	// Not guaranteed to be set for pool creation events triggered outside of FireFly
 	TransactionID *fftypes.UUID
 
+	// BlockchainTXID is the blockchain transaction hash that resulted in the event that created the pool
+	BlockchainTXID string
```

Comment:
This should not be needed (here or on TokenTransfer below) since it's been included in `blockchain.Event`